src/Functions/translate.cpp

```diff
@@ -44,14 +44,6 @@ struct TranslateImpl
             map[static_cast<unsigned char>(map_from[i])] = static_cast<UInt8>(map_to[i]);
         }
 
-        // Handle any remaining characters in map_from by assigning a default value
-        for (size_t i = min_size; i < map_from.size(); ++i)
-        {
-            if (!isASCII(map_from[i]))
-                throw Exception(ErrorCodes::BAD_ARGUMENTS, "Second argument must be ASCII strings");
-            map[static_cast<unsigned char>(map_from[i])] = ascii_upper_bound + 1;
-        }
-
```

Comment:
Why do we need to remove this part? It seems like we would need to remove the next `for` loop instead

Reply:
yes, I will change it